### PR TITLE
fix(extension-react-tables): borders should be applied for table headers on hover of column controller

### PR DESCRIPTION
**Pull request by devcer**
### Description

Added styles for `th` when column controller is hovered. i.e. preselectColumn is not -1 
fixes: https://github.com/remirror/remirror/issues/1570

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test`.

### Screenshots

Screenshot 2022-03-31 at 7 22 18 PM

### CODE REVIEW

File: packages/remirror__extension-react-tables/src/table-plugins.ts [hunk 1]
```diff
@@ -31,6 +31,9 @@ export function getTableStyle(attrs: ControllerStateValues): string {
   if (attrs.preselectColumn !== -1) {
     classNames = css`
       & table.${ExtensionTablesTheme.TABLE} tbody tr {
+        th:nth-child(${attrs.preselectColumn + 1}) {
+          ${preselectClass};
+        }
         td:nth-child(${attrs.preselectColumn + 1}) {
```
Comment (whawker) [inline packages/remirror__extension-react-tables/src/table-plugins.ts:37]: Could combine these

```suggestion
        th, td {
          &:nth-child(${attrs.preselectColumn + 1}) {
            ${preselectClass};
          }
       }
```

**Comment by whawker** [comment]
Hi @devcer thanks very much for the contribution!

**Comment by devcer** [comment]
Thank you @whawker ! I have accepted your change commit. looking forward to make more and better contributions. \o/

File: .changeset/great-squids-sleep.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@remirror/extension-react-tables': minor
```
[the remark this answers is not in the document]
Comment (devcer) [reply .changeset/great-squids-sleep.md]: Hey @whawker, I have made the change.

**Comment by whawker** [comment]
Thanks for the contribution @devcer. Really appreciated!

**Pull request merged**